web/models.py

```
@@ -2175,17 +2180,30 @@
 class UserQuiz(models.Model):
     """Model for tracking user quiz attempts and responses"""
 
+    CORRECTION_STATUS = [
+        ("not_needed", "No Correction Needed"),
+        ("pending", "Correction Pending"),
+        ("in_progress", "Correction In Progress"),
+        ("completed", "Correction Completed"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)